components/Tools/ToolDescription.tsx

```
@@ -0,0 +1,30 @@
+    <>
+      {(toolName === "getCampaign" || toolName === "getArtistAnalysis") && (
+        <>
+          {loading && !content ? (
+            <div className="flex gap-2 items-center">
+              <p>is thinking...</p>
+              <LoaderCircle className="h-4 w-4 animate-spin" />
+            </div>
+          ) : (
+            <Answer content={content} role={message.role} />
+          )}
+        </>
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)